schemas/com.snowplowanalytics.snowplow/pii_enrichment/jsonschema/1-0-0

```diff
@@ -0,0 +1,654 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a PII enrichment original values",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "pii_enrichment",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "pii": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "pojo": {
+            "type": "object",
+            "properties": {
+              "user_id": {
+                "type": "object",
+                "properties": {
+                  "originalValue": {
+                    "type": "string"
+                  },
+                  "modifiedValue": {
+                    "type": "string"
+                  },
+                  "strategy": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "enum": [
+                          "pseudonymize"
+                        ]
+                      },
+                      "function": {
+                        "enum": [
+                          "MD2",
+                          "MD5",
+                          "SHA-1",
+                          "SHA-256",
+                          "SHA-384",
+                          "SHA-512"
+                        ]
+                      }
+                    }
+                  }
+                }
+              },
+              "user_ipaddress": {
+                "type": "object",
+                "properties": {
+                  "originalValue": {
+                    "type": "string"
+                  },
+                  "modifiedValue": {
+                    "type": "string"
+                  },
+                  "strategy": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "enum": [
+                          "pseudonymize"
+                        ]
+                      },
+                      "function": {
+                        "enum": [
+                          "MD2",
+                          "MD5",
+                          "SHA-1",
+                          "SHA-256",
+                          "SHA-384",
+                          "SHA-512"
+                        ]
+                      }
+                    }
+                  }
+                }
+              },
+              "user_fingerprint": {
+                "type": "object",
+                "properties": {
+                  "originalValue": {
+                    "type": "string"
+                  },
+                  "modifiedValue": {
+                    "type": "string"
```

Comment:
I believe all `modifiedValue` should have `maxLength: 128` (SHA-512's length - longest hash of these if I'm not mistaken)